src/theme/themeProperties.ts

```diff
@@ -22,7 +22,6 @@ const themeProperties = {
   primaryColor: '#68B030',
   primaryDarkColor: '#007a49',
   primaryLightColor: '#79AE4633',
-  primaryDarkColorX: '#219653',
```

Comment:
This was removed, but it is used in a number of places within the project currently.